userspace/libsinsp/filterchecks.cpp

```diff
@@ -4101,6 +4092,16 @@ uint8_t* sinsp_filter_check_user::extract(sinsp_evt *evt, OUT uint32_t* len, boo
 		RETURN_EXTRACT_CSTR(uinfo->homedir);
 	case TYPE_SHELL:
 		RETURN_EXTRACT_CSTR(uinfo->shell);
+	case TYPE_LOGINUID:
+		RETURN_EXTRACT_VAR(tinfo->m_loginuid);
+	case TYPE_LOGINNAME:
+		ASSERT(m_inspector != NULL);
+		uinfo = m_inspector->get_user(tinfo->m_loginuid);
+		if(uinfo == NULL)
+		{
+			return NULL;
+		}
+		RETURN_EXTRACT_VAR(uinfo->name);
```

Comment:
@arossert Should this use `RETURN_EXTRACT_CSTR` instead?

Reply:
Yes, I will fix that, thanks.